lang/pt.js

```diff
@@ -68,7 +68,7 @@ export default {
 	"components.input-time-range.startTime": "Hora de início",
 	"components.interactive.instructions": "Pressione Enter para interagir, Escape para sair",
 	"components.link.open-in-new-window": "Abre em uma nova janela.",
-	"components.list.keyboard": "Use <b>arrow keys</b> to move focus inside this list, or <b>page up/down</b> to move up or down by 5",
+	"components.list.keyboard": "Use <b>teclas de seta</b> para mover o foco dentro dessa lista, ou <b>página acima/abaixo</b> para subir ou descer 5",
```

Comment:
It's interesting that "Use" wasn't translated here... looking at other similar langterms in portugese, I would expect to see a translation here...

https://github.com/BrightspaceUI/core/blob/bdff1722f4f22aec056cc1c0f896b60933221ad7/lang/en.js#L56

https://github.com/BrightspaceUI/core/blob/bdff1722f4f22aec056cc1c0f896b60933221ad7/lang/pt.js#L56

Reply:
Submitted this as an error: [TR-111](https://desire2learn.atlassian.net/servicedesk/customer/portal/38/TR-111)